persistence.js

```diff
@@ -491,7 +513,7 @@ RedisPersistence.prototype.streamWill = function (brokers) {
 }
 
 RedisPersistence.prototype.getClientList = function (topic) {
-  var clientIds = this._matcher.match(topic).map(toClientIds)
+  var clientIds = this._matcher.match(topic).filter(isTopic).map(toClientIds)
```

Comment:
I'm actually wondering whether it's better to do the filtering in the qlobber match somehow.

Btw I'm assuming that the current implementation of `getClientList` here is wrong. Unlike the implementations in the other aedes-persistences, it will returning duplicate client IDs if there is more than one match for the topic.